src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -233,50 +263,62 @@ const ScenarioParameters = ({
   const noTabsShown = parametersGroupsMetadata.length === 0;
   const isCurrentScenarioRunning = currentScenario.data.state === SCENARIO_RUN_STATE.RUNNING;
 
+  const [accordionSummaryExpanded, setAccordionSummaryExpanded] = useState(
+    localStorage.getItem('parametersAccordionExpanded') === 'true'
+  );
```

Comment:
Could you move the `useState` declaration upward, near the top of the component declaration, to be next to the other `useState`s ?